includes/utilities.inc

```diff
@@ -98,16 +98,21 @@ function islandora_ocr_get_tesseract_installed_languages($tesseract = NULL) {
  *   otherwise the abbreviation is returned unaltered.
  */
 function islandora_ocr_tesseract_language_name($language) {
-  $language_names = array(
-    'eng' => t('English'),
-    'fra' => t('French'),
-    'deu-frak' => t('German'),
-    'por' => t('Portugese'),
-    'spa' => t('Spanish'),
-    'hin' => t('Hindi'),
-    'jpn' => t('Japanese'),
-    'ita' => t('Italian'),
-  );
+    $language_names = module_invoke_all('get_ocr_tesseract_languages');
```

Comment:
Based on the comment I left in the `api.php` document, you'll also need to change the name of the hook here.